tests/functional/newsletter/test_newsletter_embed.py

```diff
@@ -58,6 +61,9 @@ def test_newsletter_default_values(page_class, url_kwargs, base_url, selenium):
     assert page.newsletter.is_privacy_policy_link_displayed
 
 
+# /firefox Quantum page behind switch, unskip when the switch is removed
+@pytest.mark.skipif(
```

Comment:
Instead of skipping all the tests, skip just the failing one:

```python
@pytest.mark.parametrize('page_class', [
    HomePage,
    ContributePage,
    pytest.mark.skipif(HomePage, reason='https://bugzilla.mozilla.org/show_bug.cgi?id=1293779')])
```